.storybook/themes/commerce.scss

```diff
@@ -456,6 +456,33 @@
 				),
 			),
 			blocks: (
+				alert-bar: (
+					align-items: center,
+					background-color: var(--background-color-danger),
```

Comment:
The background colour is different to the existing implementation and designs. Can we use the `red-4`